### ISSUE TITLE
is everything okay? 

### ISSUE THREAD

**Issue by pavankatari** [longjunyu2/winlator#165]
Hello Longjunyu  how are you ? From last few months no updates, last updated in August , hope all good, I like you project i hope you will continue project good luck

**Comment by lvonasek** [comment]
I am bit worry too as I didn't hear from @longjunyu2 longer time.

In the worst case scenario, I could release a new version. But that would be incompatible with the current version (due to different signing key). That means people would lose their existing data (which I would like to avoid).

**Comment by pavankatari** [comment]
It's okay release, i think people knows how to backup game save files , so please release 😅

**Comment by lvonasek** [comment]
> It's okay release it, i think people knows how to backup game save files , so please release 😅

OK https://github.com/longjunyu2/winlator/releases/tag/v7.1.4

**Comment by pavankatari** [comment]
> > It's okay release it, i think people knows how to backup game save files , so please release 😅
> 
> OK https://github.com/longjunyu2/winlator/releases/tag/v7.1.4

You merged code-refactor branch ? I mean you merged all latest commits ?

**Comment by lvonasek** [comment]
No, code-refactor isn't ready. I build it from the main branch.

**Comment by pavankatari** [comment]
> No, code-refactor isn't ready. I build it from the main branch.

Longjunyu from china ? He don't use any social media? I mean telegram or discord?

**Comment by lvonasek** [comment]
Yes, I am not aware of any his social account.

**Comment by pavankatari** [comment]
> Yes, I am not aware of any his social account.

Maybe he is working on game fusion 😅

**Comment by K11MCH1** [comment]
Longjunyu has been dealing with studies and other projects up till now. Glad hes back!

**Comment by pavankatari** [comment]
> Longjunyu has been dealing with studies and other projects up till now. Glad hes back!

You have his any contact?

**Comment by lvonasek** [comment]
His contact is 877730493@qq.com. I didn't contact him. He gets a lot of email notifications from the repository itself.
